### DISCUSSION TITLE
Don't know how to get code lens working

### DISCUSSION THREAD

**Discussion by ByteDrummer** [neoclide/coc.nvim#3627] [Q&A] [answered]
I have code lens enabled in my `coc-settings.json`, my Neovim version is `v0.7.0-dev+1027-gf02a5a7bd`, and I'm using `coc-tsserver`. I don't see any virtual text for code lens on JS files. Can anyone help me understand what's going on? Should I be using a different extension for this feature?

**Comment by chemzqm** [comment, marked as answer]
Check the log to make sure you have codeLens request and response: https://github.com/neoclide/coc-tsserver#troubleshooting
You may need:
```
  "javascript.referencesCodeLens.enable": true,
  "javascript.referencesCodeLens.showOnAllFunctions": true,
```
configurations.

**Reply by ByteDrummer** [reply]
Thank you. `"javascript.referencesCodeLens.showOnAllFunctions": true` is what I needed.
